common/src/main/java/com/microsoft/identity/common/internal/cache/BrokerOAuth2TokenCache.java

```diff
@@ -123,15 +146,15 @@
     public BrokerOAuth2TokenCache(@NonNull final Context context,
                                   int callingProcessUid,
                                   @NonNull IBrokerApplicationMetadataCache applicationMetadataCache) {
-        mContext = context;
+        super(AndroidPlatformComponents.createFromContext(context));
```

Comment:
this(AndroidPlatformComponents.createFromContext(context), callingProcessUid, applicationMetadataCache)

Reply:
(applicable to any function that takes in a context object)